spaces/S000210/README.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: S000210
+name: Isbell's locally compact non-k-space topology on $\omega_1+1$
+refs:
+  - doi: 10.1090/S0002-9939-1987-0891170-3
+    name: A distinguishing example in k-spaces (Isbell)
+  - mathse: 3904987
+    name: Answer to "Is there a locally compact space which is not a k-space"
+---
+
+The set $\omega_1+1$ generated by the basis
```

Comment:
As written, that makes no sense.  You meant "the set ... with the topology generated by ...", right?

Reply:
```suggestion
The set $\omega_1+1$ with the topology generated by the basis
```